src/modules/apiData.js

```diff
@@ -0,0 +1,4 @@
+const apiData = 'https://www.themealdb.com/api/json/v1/1/filter.php?a=American';
+const likeUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/udgMt8FzeFXMHjSp0LMg/likes/';
+const commentUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/udgMt8FzeFXMHjSp0LMg/comments/';
```

Comment:
Please, kindly use Descriptive Variable Names: While apiData, likeUrl, and commentUrl are descriptive enough, you could make them even more meaningful by using longer, more descriptive names, such as mealApiUrl, likeApiUrl, and commentApiUrl. This can make your code more self-explanatory for anyone reading it.

Please Consider adding comments to explain the purpose of these URLs and how they are used other developers will be working with your code. Additionally, you could document the expected behavior of these APIs, expected response formats, and any special considerations. Thanks 😄